python/subunit/__init__.py

```diff
@@ -575,7 +575,7 @@ def _handleTime(self, offset, line):
             event_time = iso8601.parse_date(line[offset:-1].decode())
         except TypeError:
             raise TypeError(_u("Failed to parse %r, got %r")
-                % (line, sys.exec_info[1]))
+                % (line, sys.exc_info()[1]))
```

Comment:
I think we could probably replace this with:

```
except TypeError as e:
    raise TypeError(_u("Failed to parse %r, got %r") % (line, e)))
```

but this is obviously also correct and what was there didn't work, so lets just make the smaller fix